android/src/main/cpp/NativeProxy.cpp

```diff
@@ -38,6 +38,8 @@ NativeProxy::NativeProxy(
 #ifdef RCT_NEW_ARCH_ENABLED
     jni::alias_ref<facebook::react::JFabricUIManager::javaobject>
         fabricUIManager,
+    std::function<void(std::function<void(jsi::Runtime &runtime)> &&callback)>
+        runtimeExecutor,
```

Comment:
1. Shouldn't we also wrap this in `RCT_NEW_ARCH_ENABLED`?
2. RuntimeExecutor is available only on Fabric or can we also use it on Paper? I would like to use RuntimeExecutor in favor of JSCallInvoker wherever possible.

Reply:
1. This is (was) wrapped in `RCT_NEW_ARCH_ENABLED`.

2. It doesn't seem to be working on Paper.